### PR TITLE
Doc/edits general doc monday pm

### PR DESCRIPTION
**Pull request by PipKat**
Past quitting time! There's always tomorrow, right?

### CODE REVIEW

File: pyaedt/application/AnalysisRMxprt.py [hunk 1]
```diff
@@ -112,15 +102,15 @@ def enable_modelcreation(self, solution_type = None):
...
+        """Check solution consistency."""
         if self._solution_type:
             return self._odesign.GetSolutionType() == self._solution_type
         else:
             return True
 
     @aedt_exception_handler
     def _check_design_consistency(self):
-        """ """
+        """Check design consitency."""
```
Comment (PipKat) [inline pyaedt/application/AnalysisRMxprt.py:113]:
```suggestion
        """Check design consistency."""
```

File: pyaedt/application/Analysis.py [hunk 2]
```diff
@@ -789,11 +744,11 @@ def get_object_material_properties(self, object_list=None, prop_names=None):
         Parameters
         ----------
         object_list : list, optional
-            List of objects for which to get material_properties. The default is ``None``. 
-            If ``None``, all objects are considered. 
+            List of objects for which to get material_properties. The default is ``None``,  
+            in wihch case all objects are considered.
```
Comment (PipKat) [inline pyaedt/application/Analysis.py:748]:
```suggestion
            in which case all objects are considered. 
```

File: pyaedt/application/Analysis.py [hunk 3]
```diff
@@ -26,15 +26,16 @@
...
+    """Contains all common analysis functions.
     
     It is automatically initialized by a call from an application, such as HFSS or Q3D. 
     See the application function for its parameter descriptions.
 
     Parameters
     ----------
+    Design :
+    
+    object :
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:38]:
```suggestion
    application : str
        Application that is to initialize the call.
    projectname : str
        Name of the project to select or the full path to the project
        or AEDTZ archive to open.
    designname : str
        Name of the design to select.
    solution_type : str
        Solution type to apply to the design.
    setup_name : str
        Name of the setup to use as the nominal.
    specified_version: str
        Version of AEDT  to use.
    NG : bool
        Whether to run AEDT in the non-graphical mode. 
    AlwaysNew : bool
        Whether to launch an instance of AEDT in a new thread, even if
        another instance of the ``specified_version`` is active on the
        machine.
    release_on_exit : bool
        Whether to release  AEDT on exit.
    student_version : bool
        Whether to enable the student version of AEDT. 
```
The `Design` and `object` shown in the Analysis class as `Analysis(Design, object)` mean that the class inherits from the `Design` and `object` classes. For the class parameters, the parameters for the `__init__` method is what matters.

File: pyaedt/application/Analysis.py [hunk 4]
```diff
@@ -63,16 +64,12 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
 
     @property
     def materials(self):
-        """Material Manager that is used to manage materials in the project.
+        """Material manager for managing materials in the project.
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:67]:
```suggestion
        """Manages materials in the project.
```
"Material manager for managing materials" sounded a bit wordy to me. Feel free to ignore this suggestion.

File: pyaedt/application/Analysis.py [hunk 5]
```diff
@@ -63,16 +64,12 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
...
         
-        Parameters
-        ----------
-
         Returns
         -------
         :class:`pyaedt.modules.MaterialLib.Materials`
-            Material Manager that is used to manage materials in the project.
-           
+            Material manager for managing materials in the project.
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:72]:
```suggestion
            Manages materials in the project.
```
Same comment as above. Feel free to ignore this.

File: pyaedt/application/Analysis.py [hunk 6]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
+        """Available variation object.
         
-        :return: Available Variation Object
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:96]:
```suggestion
        :class:`pyaedt.application.Analysis.Analysis.AvailableVariations`
```

File: pyaedt/application/Analysis.py [hunk 7]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
+        """Coordinate system axis constant.
         
-        :return: Coordinate System Axis Constants Tuple (.X, .Y, .Z)
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:108]:
```suggestion
        tuple
```

File: pyaedt/application/Analysis.py [hunk 8]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
+            Coordinate System Axis Constants Tuple (.X, .Y, .Z).
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:109]:
```suggestion
            Coordinate system axis constants tuple (.X, .Y, .Z).
```

File: pyaedt/application/Analysis.py [hunk 9]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
         
-        :return: Coordinate System Plane Constants Tuple(.XY, .YZ, .XZ)
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
+            Coordinate system plane constants tuple(.XY, .YZ, .XZ).
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:121]:
```suggestion
            Coordinate system plane constants tuple (.XY, .YZ, .XZ).
```

File: pyaedt/application/Analysis.py [hunk 10]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
+        """Coordinate system plane constants.
         
-        :return: Coordinate System Plane Constants Tuple(.XY, .YZ, .XZ)
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:120]:
```suggestion
        tuple
```

File: pyaedt/application/Analysis.py [hunk 11]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
 
         """
         return CoordinateSystemPlane()
 
     @property
     def View(self):
-        """Planes To check if redundant to CoordinateSystemPlane
-        
+        """Planes. (To check if redundant to CoordinateSystemPlane.)
+
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:129]:
```suggestion
       
```

File: pyaedt/application/Analysis.py [hunk 12]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
         
-        :return: Coordinate System Plane String Tuple("XY", "YZ", "XZ")
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
+            Coordinate System Plane String Tuple("XY", "YZ", "XZ").
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:134]:
```suggestion
        tuple
            Coordinate system plane string tuple ("XY", "YZ", "XZ").
```

File: pyaedt/application/Analysis.py [hunk 13]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
+        """Gravity direction. (To check if redundant.)
         
-        :return: Gravity Direction Tuple (XNeg, YNeg, ZNeg, XPos, YPos, ZPos)
-
-        Parameters
-        ----------
-
         Returns
         -------
+        Gravity direction tuple (XNeg, YNeg, ZNeg, XPos, YPos, ZPos).
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:145]:
```suggestion
        tuple
            Gravity direction tuple (XNeg, YNeg, ZNeg, XPos, YPos, ZPos).
```

File: pyaedt/application/Analysis.py [hunk 14]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
         
-        :return: mesh object
-
-        Parameters
-        ----------
-
         Returns
         -------
-
+        type
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:167]:
```suggestion
        :class:`pyaedt.modules.Mesh.Mesh`
```

File: pyaedt/application/Analysis.py [hunk 15]
```diff
@@ -95,148 +89,122 @@ def Position(self):
...
+        """Solution.
         
-        :return: Solutions Module object
-
-        Parameters
-        ----------
-
         Returns
         -------
+        :class:`pyaedt.modules.AdvancedPostProcessing.PostProcessor.post_osolution`
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:189]:
```suggestion
        AEDT object
```
This is a call made to the AEDT application to get the Solutions object to then perform some operations. There really isn't a type for this object, so I wrote "AEDT object" but feel free to change it.

File: pyaedt/application/Analysis.py [hunk 16]
```diff
@@ -317,14 +270,13 @@ def nominal_adaptive(self):
...
+        """Nominal sweep.
 
         Returns
         -------
         str
-            sweep is available it will return the lastadaptive
-            
-            :return: str nominal setup sweep if present
+            Name of the last adaptive if a sweep is available or 
+            the name of the nominal setup sweep if present.
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:279]:
```suggestion
            the name of the nominal adaptive sweep if present.
```

File: pyaedt/application/Analysis.py [hunk 17]
```diff
@@ -334,50 +286,38 @@ def nominal_sweep(self):
...
-
-        Parameters
-        ----------
-
+        """Output variables.
+     
         Returns
         -------
+        type
+            Solutions output variable object.
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:307]:
```suggestion
        list
            List of output variables.
```

File: pyaedt/application/Analysis.py [hunk 18]
```diff
@@ -387,80 +327,94 @@ def setup_names(self):
...
         
-        :return: Solutions Optimetrics object
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
+            Solutions optimetrics object.
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:335]:
```suggestion
        AEDT object
            Optimetrics object.
```

File: pyaedt/application/Analysis.py [hunk 19]
```diff
@@ -387,80 +327,94 @@ def setup_names(self):
...
         
-        :return: Solutions OutputVariable object
-
-        Parameters
-        ----------
-
         Returns
         -------
+        type
+            Solutions output variable object/
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:347]:
```suggestion
        AEDT object
            Output variable object.
```

File: pyaedt/application/Analysis.py [hunk 20]
```diff
@@ -387,80 +327,94 @@ def setup_names(self):
...
 
             Parameters
             ----------
             parent :
-                parent object
+                Inherited parent object.
 
             Returns
             -------
+            type
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:388]:
```suggestion
            object
```

File: pyaedt/application/Analysis.py [hunk 21]
```diff
@@ -531,31 +485,32 @@ def all(self):
...
 
     @aedt_exception_handler
     def get_nominal_variation(self):
-        """:return: nominal variation"""
+        """Retrieve the nominal variation.
+        
+        Returns
+        -------
+        type
+            Nominal variation.
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:512]:
```suggestion
        list
            List of nominal variations.
```

File: pyaedt/application/Analysis.py [hunk 22]
```diff
@@ -654,21 +609,21 @@ def generate_unique_setup_name(self, setup_name=None):
...
-            default type is applied.
+            Type of the setup. The default is ``None``, in which case 
+            the default type is applied.
         props : dict, optional
             Dictionary of properties with values. The default is ``{}``.
 
         Returns
         -------
-        Setup
+        type
```
Comment (jleonatti) [inline pyaedt/application/Analysis.py:626]:
```suggestion
        :class:`pyaedt.modules.SolveSetup.Setup`
```

File: pyaedt/application/Analysis2D.py [hunk 23]
```diff
@@ -7,11 +7,9 @@
 
 
 class FieldAnalysis2D(Analysis):
-    """**FieldAnalysis2D class.**
+    """**Manages 2D field analysis setup in Maxwell2D and Q2D.
```
Comment (jleonatti) [inline pyaedt/application/Analysis2D.py:10]:
```suggestion
    """Manages 2D field analysis setup in Maxwell2D and Q2D.
```

File: pyaedt/application/Analysis3D.py [hunk 24]
```diff
@@ -300,7 +297,7 @@ def assignmaterial(self, obj, mat):
            Use :func:`FieldAnalysis3D.assign_material` instead.
 
         """
-        warnings.warn('assignmaterial is deprecated.  Please use assign_material instead.',
+        warnings.warn('assignmaterial is deprecated.  Use assign_material instead.',
```
Comment (jleonatti) [inline pyaedt/application/Analysis3D.py:300]:
```suggestion
        warnings.warn('assignmaterial is deprecated. Use assign_material instead.',
```

File: pyaedt/application/Analysis3DLayout.py [hunk 25]
```diff
@@ -65,12 +63,18 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
 
     @property
     def oboundary(self):
-        """Boundary object."""
+        """Boundary.
+        
+        Returns
+        -------
+        :class:`pyaedt.modeler.Object3d.Object3d`
```
Comment (jleonatti) [inline pyaedt/application/Analysis3DLayout.py:70]:
```suggestion
        AEDT object
            Boundaries object.
```

File: pyaedt/application/AnalysisIcepak.py [hunk 26]
```diff
@@ -236,7 +239,7 @@ def assignmaterial(self, obj, mat):
 
         """
         # raise a DeprecationWarning.  User won't have to change anything
-        warnings.warn('assignmaterial is deprecated.  Please use assign_material instead.',
+        warnings.warn('assignmaterial is deprecated.  Use assign_material instead.',
```
Comment (jleonatti) [inline pyaedt/application/AnalysisIcepak.py:242]:
```suggestion
        warnings.warn('assignmaterial is deprecated. Use assign_material instead.',
```

File: pyaedt/application/AnalysisRMxprt.py [hunk 27]
```diff
@@ -5,14 +5,10 @@
...
 class FieldAnalysisRMxprt(Analysis):
-    """**AEDT_RMxprtAnalysis**
-    
-    
-    To BE Implemented
-    Class for RMXpert Field Analysis Setup
-    
-    It is automatically initialized by Application call (like HFSS,
-    Q3D...). Refer to Application function for inputs definition
+    """Manages RMXpert field analysis setup. (To be implemented.)
```
Comment (jleonatti) [inline pyaedt/application/AnalysisRMxprt.py:8]:
```suggestion
    """Manages RMXprt field analysis setup. (To be implemented.)
```

File: pyaedt/application/AnalysisRMxprt.py [hunk 28]
```diff
@@ -61,22 +45,28 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
 
     @property
     def modeler(self):
-        """ """
+        """Modeler.
+        
+        Returns
+        -------
+      	:class:`pyaedt.modules.Modeler`
```
Comment (jleonatti) [inline pyaedt/application/AnalysisRMxprt.py:52]:
```suggestion
        Returns
        -------
        :class:`pyaedt.modules.Modeler`
```

File: pyaedt/application/AnalysisRMxprt.py [hunk 29]
```diff
@@ -61,22 +45,28 @@ def __init__(self, application, projectname, designname, solution_type, setup_na
...
     @aedt_exception_handler
     def disable_modelcreation(self, solution_type=None):
-        """It Enables RMxprtSolution
+        """Enable the RMxprt solution.
 
         Parameters
         ----------
-        solution_type :
-            solutionType (Default value = None)
+        solution_type :
```
Comment (jleonatti) [inline pyaedt/application/AnalysisRMxprt.py:63]:
```suggestion
        solution_type :
```

File: pyaedt/application/AnalysisRMxprt.py [hunk 30]
```diff
@@ -86,17 +76,17 @@ def disable_modelcreation(self, solution_type=None):
 
     @aedt_exception_handler
     def enable_modelcreation(self, solution_type = None):
-        """It Enables ModelCreation  for Maxwell Model Wizard
+        """Enable model creation for the Maxwell model wizard.
 
         Parameters
         ----------
         solution_type :
```
Comment (jleonatti) [inline pyaedt/application/AnalysisRMxprt.py:83]:
```suggestion
        solution_type : str
```

File: pyaedt/application/Design.py [hunk 31]
```diff
@@ -645,7 +647,7 @@ def design_type(self):
         Returns
         --------
         str
-            Design type.  See above for one of the possible return values.
+            Type of the design type. See above for a list of possible return values.
```
Comment (jleonatti) [inline pyaedt/application/Design.py:650]:
```suggestion
            Type of the design. See above for a list of possible return values.
```

**Review by jleonatti — APPROVED** [review]

File: pyaedt/application/Analysis.py (hunk 3 above)
Comment (PipKat) [reply pyaedt/application/Analysis.py:38]: @jleonatti Thanks, Jess. I didn't know this!

File: pyaedt/application/Analysis.py (hunk 4 above)
Comment (PipKat) [reply pyaedt/application/Analysis.py:67]: @jleonatti Totally agree. I just wasn't sure who wrote the original and didn't want to offend them!

File: pyaedt/desktop.py [hunk 32]
```diff
@@ -143,7 +143,7 @@ def release_desktop(close_projects=True, close_desktop=True):
     Parameters
     ----------
     close_projects : bool, optional
-        Whether to close the projects opened in the session. The default is ``True``.
+        Whether to close the AEDT projects opened in the session. The default is ``True``.
```
Comment (PipKat) [inline pyaedt/desktop.py:146]:
```suggestion
        Whether to close the AEDT projects open in the session. The default is ``True``.
```
